model/src/main/java/com/basistech/rosette/dm/TranslatedData.java

```diff
@@ -30,26 +30,63 @@
 @EqualsAndHashCode(callSuper = true)
 public class TranslatedData extends BaseAttribute implements Serializable {
     private static final long serialVersionUID = 250L;
+    private final TextDomain sourceDomain;
+
+    /**
+     * @deprecated Replaced by targetDomain
+     */
+    @Deprecated(forRemoval = true)
```

Comment:
Since we can use `JsonAlias` to have a single json field for the target domain, we should also be able to completely replace `domain` with `targetDomain` in this class, and only leave the old getter marked as deprecated.
We also (I think) want to deprecate the old constructor, since we don't want to encourage carrying around null sourceDomain information